components/CreateCycle/DraggableWorkoutList.jsx

```diff
@@ -18,6 +18,7 @@ const DraggableWorkoutList = (props) => {
   const renderCard = ({ item, index, drag }) => (
     <ExpandableWorkoutCard
       drag={drag}
+      removeWorkout={removeWorkout}
```

Comment:
If you swap removeWorkout to take in the index of the workout you can pass it here, you'll need to use currying but I can help you out with that if you need help